dart/lib/src/sentry_client_attachment_processor.dart

```diff
@@ -0,0 +1,11 @@
+import 'dart:async';
+
+import './sentry_attachment/sentry_attachment.dart';
+import './protocol/sentry_event.dart';
+
+class SentryClientAttachmentProcessor {
```

Comment:
This should be `@internal`

Reply:
Can't export and use this in the flutter SDK if marked internal i think. At least `flutter analyze` is complaining.

Reply:
we can use the ignore linter for this use case, as we do with some other classes when needed, does that work?

Reply:
added multiple ignores